02-getting-started.Rmd

```diff
@@ -45,7 +45,7 @@ For much of this book, we will assume that you are using R via RStudio. First ti
 
 R: Engine            |  RStudio: Dashboard 
 :-------------------------:|:-------------------------:
-![](images/engine.jpg){ height=1.7in }  |  ![](images/dashboard.jpg){ height=1.7in }
+![](images/ss/319540715.png){ height=1.7in }  |  ![](images/ss/228620182.png){ height=1.7in }
```

Comment:
Could you find a dashboard image with a steering wheel and perhaps the gear shift? The idea being you control the engine thru the steering wheel and gear shift.

Reply:
ID  379218673